### DISCUSSION TITLE
Migrate from 1.7.1 to 2.2.4

### DISCUSSION THREAD

**Discussion by welfendrossel** [jquery/jquery-migrate#485] [Q&A]
Hello
I'm not sure how to interpret this:
---------------------------------------------------------------------------------------------------------------------------
jquery-migrate-1.4.1.js:23 JQMIGRATE: Migrate is installed with logging active, version 1.4.1
jquery-migrate-1.4.1.js:45 JQMIGRATE: jQuery.fn.load() is deprecated
migrateWarn @ jquery-migrate-1.4.1.js:45
jQuery.fn.<computed> @ jquery-migrate-1.4.1.js:560
events @ foundation.topbar.js:248
bind @ foundation.js:115
each @ jquery-2.2.4.js:365
each @ jquery-2.2.4.js:137
bindings @ foundation.js:122
init @ foundation.topbar.js:28
init_lib @ foundation.js:367
init @ foundation.js:334
(anonymous) @ foundation.js:720
each @ jquery-2.2.4.js:365
each @ jquery-2.2.4.js:137
$.fn.foundation @ foundation.js:719
(anonymous) @ Events:7900
jquery-migrate-1.4.1.js:47 console.trace
migrateWarn @ jquery-migrate-1.4.1.js:47
jQuery.fn.<computed> @ jquery-migrate-1.4.1.js:560
events @ foundation.topbar.js:248
bind @ foundation.js:115
each @ jquery-2.2.4.js:365
each @ jquery-2.2.4.js:137
bindings @ foundation.js:122
init @ foundation.topbar.js:28
init_lib @ foundation.js:367
init @ foundation.js:334
(anonymous) @ foundation.js:720
each @ jquery-2.2.4.js:365
each @ jquery-2.2.4.js:137
$.fn.foundation @ foundation.js:719
(anonymous) @ Events:7900
---------------------------------------------------------------------------------------
Is this some kind of stack trace? That last line is my code.
When I click on it I got this:
     <script>
        $(document).foundation();
    </script>

![image](https://user-images.githubusercontent.com/53899900/222672615-f66a7db5-862a-4896-8172-6177b75257a3.png)

Kind regards
/mique

**Comment by mgol** [comment]
You can find the meaning of all warnings at https://github.com/jquery/jquery-migrate/blob/main/warnings.md or https://github.com/jquery/jquery-migrate/blob/1.x-stable/warnings.md depending on the Migrate version used. The cause of the warning may come from your code but it may also come from a library you're using, which is likely the cause here.
